include/llvm-dialects/Dialect/Visitor.h

```diff
@@ -123,7 +123,9 @@ class VisitorKey {
   unsigned m_intrinsicId = 0;
 };
 
-struct VisitorCallbackData {
+class Foo0 {};
```

Comment:
> Can you add some more context in the code on why this is necessary + give some better naming to the dummy classes than `Foo_i`?

Add new comment to the code.  Since it is dummy class.  i think Foo is good name to me or if you have any suggestions?

Reply:
I think it is sufficient if there is some explanation on what you are using the classes for